imod_coupler/drivers/driver.py

```diff
@@ -9,6 +11,26 @@
 from imod_coupler.config import BaseConfig
 
 
+def resolve_path(libname: str) -> str:
+    match sys.platform.lower():
+        case "win32":
+            env_var = "PATH"
+        case "linux" | "linux2" | "darwin":
+            env_var = "LD_LIBRARY_PATH"
+        case _:
+            return libname
+
+    if os.path.isfile(libname):
+        return libname
+    if env_var in os.environ:
+        pathdef: str = os.environ[env_var]
+        for dir in pathdef.split(os.pathsep):
+            full_path = Path(dir) / libname
+            if os.path.isfile(full_path):
```

Comment:
You can just use `full_path.is_file()` here right? Since `is_file()` is a method on Path objects.